spec/mailers/registration_mailer_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "spec_helper"
+
+  RSpec.describe RegistrationMailer do
+    describe 'notified registration' do
+    let(:user) { FactoryGirl.create(:donor_user) }
```

Comment:
indentation - needs to be pushed out since it's inside the `describe 'notified registration' do` block